ceph.spec.in

```diff
@@ -153,6 +153,7 @@ BuildRequires:  libopenssl-devel
 BuildRequires:  lsb-release
 BuildRequires:  openldap2-devel
 BuildRequires:	python-Cython
+BuildRequires:  python-setuptools
```

Comment:
@smithfarm we have
```
%if 0%{?rhel}
BuildRequires:	python34-devel
BuildRequires:	python34-setuptools
BuildRequires:	python34-Cython
%else
BuildRequires:	python3-devel
BuildRequires:	python3-setuptools
BuildRequires:	python3-Cython
%endif
```

so if `python3-setuptools` depends on `python{3}-six`, we are already ready?